Fix spelling

vai-airbyte opened the pull request:
Corrected spelling errors in the document.

vai-airbyte committed:
Fix spelling 

Corrected spelling errors in the document and updated section titles.

vai-airbyte commented on docs/integrations/enterprise-connectors/source-netsuite-enterprise.md:
```diff
@@ -36,7 +36,7 @@ To find details such as host, port, Account ID and role go on Netsuite home page
 
 ### Authentication Methods
-Source Netsuite supports all authentication methods in SuiteAnalytics connect: username and password, Token Based Authentication (TBA) and OAuth 2.0 based authenticaiton.
+Source Netsuite supports all authentication methods in SuiteAnalytics connect: username and password, Token Based Authentication (TBA) and OAuth 2.0 based authentication.
```
I think we should capitalize it to NetSuite throughout.

ian-at-airbyte reviewed (approved):
LGTM. The linter made some suggestions which look correct, except the spelling issue. Regarding that, we can add "NetSuite" to `docs/vale-styles/config/vocabularies/Airbyte/accept.txt` and it will accept that as the correct spelling.

vai-airbyte committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

vai-airbyte committed:
Fix spelling of 'Netsuite' to 'NetSuite'

Corrected spelling of 'Netsuite' to 'NetSuite' throughout the document for consistency.

vai-airbyte reviewed (commented):
Implemented the remaining suggestions.

vai-airbyte committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

vai-airbyte committed:
Add NetSuite to vocabulary acceptance list
